CiscoMeraki/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:README.md]
```

Comment:
Why do we need this?

Reply:
In previous PRs, we received your comments to add this file.
/.pack-ignore
  See - Cisco SMA
The need for this is at your discretion.